api/api.go

```diff
@@ -217,6 +223,21 @@ func (a *API) Store(ctx context.Context, data io.Reader, size int64, toEncrypt b
 // Resolve a name into a content-addressed hash
 // where address could be an ENS name, or a content addressed hash
 func (a *API) Resolve(ctx context.Context, address string) (storage.Address, error) {
+	// if address is .rsk, resolve it with RNS resolver
+	eTLD, _ := publicsuffix.PublicSuffix(address)
```

Comment:
Public suffix list does not contain `rsk` tld, so it would be just ok to get domain tld with strings package functions, for example.

Reply:
@santicomp2014 has removed this, thanks!

Reply:
Fixed, we used the strings package as you mentioned, one less dependency to use.